src/main/java/model/LottoGenerator.java

```diff
@@ -0,0 +1,53 @@
+package model;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoGenerator {
+
+    public static final int VALID_MIN_LOTTO_NUMBER = 1;
+    public static final int VALID_MAX_LOTTO_NUMBER = 45;
+    public static final int LOTTO_NUMBERS_COUNT = 6;
+
+    public Lotto generateLotto() {
+        List<Integer> numbers = new ArrayList<>();
+
+        for (int i = VALID_MIN_LOTTO_NUMBER; i <= VALID_MAX_LOTTO_NUMBER; i++) {
+            numbers.add(i);
+        }
```

Comment:
매번 1 ~ 45 까지의 숫자를 생성해서 로또 객체를 만들 필요 없이, 캐싱해둔 로또숫자 객체들을 활용해보면 어떨까요?  😃

```suggestion
        List<LottoNumber> lottoNumbers = new ArrayList<>(LottoNumber.LOTTO_NUMBERS);
        Collections.shuffle(lottoNumbers);

        lottoNumbers = lottoNumbers.subList(0, LOTTO_NUMBERS_COUNT);
        Collections.sort(lottoNumbers);

        return new Lotto(lottoNumbers);
```

Reply:
캐싱한 숫자를 사용하도록 변경하였습니다. [109f710](https://github.com/next-step/java-lotto-kakao/pull/36/commits/109f710aaac3259966cb47766db6941f1effe6a5)